src/hooks/useSwapCurrencyList.ts

```diff
@@ -94,6 +106,20 @@ const useSwapCurrencyList = (
   const [lowLiquidityAssets, setLowLiquidityAssets] = useState<RT[]>([]);
   const [verifiedAssets, setVerifiedAssets] = useState<RT[]>([]);
 
+  const { inputCurrency } = useSwapCurrencies();
+  const crosschainSwapsEnabled = useExperimentalFlag(CROSSCHAIN_SWAPS);
+  const inputChainId = useMemo(
+    () =>
+      inputCurrency?.network &&
+      ethereumUtils.getChainIdFromNetwork(inputCurrency?.network as Network),
```

Comment:
oh i think some tokens don't have the network param and we should use type instead

i believe there's a method called ethereumUtils.getChainIdFromType to do the same

Reply:
cross commenting prs https://github.com/rainbow-me/rainbow/pull/4221/files#r982967221

Reply:
based on the cross comment, are you fine with leaving this as is? Just want to confirm.